crates/web-plugins/oob-messages/src/models.rs

```diff
@@ -74,20 +79,21 @@ impl OobMessage {
         }
     }
 
-    fn serialize_oob_message(oob_message: &OobMessage, url: &str) -> Result<String, String> {
+    fn serialize_oob_message(oob_message: &OobMessage, url: &str) -> Result<Ooburl, String> {
         let plaintext =
             to_string(oob_message).map_err(|e| format!("Serialization error: {}", e))?;
         let encoded_jwm = Base64Url.encode(plaintext.as_bytes());
 
-        Ok(format!("{}?_oob={}", url, encoded_jwm))
+        let ooburl = format!("{}/_oob{}", url, encoded_jwm);
```

Comment:
You changed the format of the oob_url. Is there any reason to that?

Reply:
yes i want to use the oob jwm as a routing path to the out band message and the previous format contains query parameters which will be misinterpreted if used as route path